cmd/celestia/rpc.go

```diff
@@ -61,13 +64,34 @@ func init() {
 		false,
 		"Print JSON-RPC request along with the response",
 	)
+	blobCmd.AddCommand(getCmd, getAllCmd, submitCmd, getProofCmd)
```

Comment:
Nit: do registering of blob sub commands in init defined next to the blob cmd 

This way it'c clearer to see and control the relation between those

Reply:
done